### ISSUE TITLE
Pkg.build"WebIO") fails, in Julia 0.7 beta2

### ISSUE THREAD

**Issue by halars** [JuliaGizmos/WebIO.jl#164]
Pkg.build("WebIO") fails with the following error:

```
 Building WebIO → `~/.julia/packages/WebIO/8IdD/deps/build.log`
┌ Error: Error building `WebIO`:
│ WARNING: Base.Pkg is deprecated, run `using Pkg` instead
│   likely near /root/.julia/packages/WebIO/8IdD/deps/build.jl:36
│ WARNING: Base.Pkg is deprecated, run `using Pkg` instead
│   likely near /root/.julia/packages/WebIO/8IdD/deps/build.jl:36
│ WARNING: Base.Pkg is deprecated, run `using Pkg` instead
│   likely near /root/.julia/packages/WebIO/8IdD/deps/build.jl:36
│ WARNING: Base.Pkg is deprecated, run `using Pkg` instead
│   likely near /root/.julia/packages/WebIO/8IdD/deps/build.jl:36
│ in install_ijulia_config at /root/.julia/packages/WebIO/8IdD/deps/build.jl
│ ┌ Warning: `replace(s::AbstractString, pat, f)` is deprecated, use `replace(s, pat => f)` instead.
│ │   caller = install_ijulia_config() at build.jl:15
│ └ @ Main ~/.julia/packages/WebIO/8IdD/deps/build.jl:15
│ ERROR: LoadError: MethodError: no method matching dir()
│ Closest candidates are:
│   dir(!Matched::String, !Matched::String...) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/Pkg/src/API.jl:433
│ Stacktrace:
│  [1] install_ijulia_config() at /root/.julia/packages/WebIO/8IdD/deps/build.jl:17
│  [2] top-level scope at none:0
│  [3] include at ./boot.jl:317 [inlined]
│  [4] include_relative(::Module, ::String) at ./loading.jl:1034
│  [5] include(::Module, ::String) at ./sysimg.jl:29
│  [6] include(::String) at ./client.jl:393
│  [7] top-level scope at none:0
│ in expression starting at /root/.julia/packages/WebIO/8IdD/deps/build.jl:36
└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/Pkg/src/Operations.jl:1007
```
Looking at deps/build.jl, Pkg.dir() is used, but is undefined in Julia 0.7:
```
julia> Pkg.dir()
ERROR: MethodError: no method matching dir()
Closest candidates are:
  dir(::String, ::String...) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v0.7/Pkg/src/API.jl:433
Stacktrace:
 [1] top-level scope at none:0
```
https://github.com/JuliaLang/Pkg.jl/issues/417

**Comment by rdeits** [comment]
Thanks for reporting. This should be fixed in https://github.com/JuliaGizmos/WebIO.jl/pull/157 which is still blocked by some breaking changes in Mux.jl.

**Comment by lewisl** [comment]
Still a problem in shipping 0.7

```
Error: Error building `WebIO`: 
│ WARNING: Base.Pkg is deprecated, run `using Pkg` instead
│   likely near /Users/lewis/.julia/packages/WebIO/7rrhi/deps/build.jl:55
│ WARNING: Base.Pkg is deprecated, run `using Pkg` instead
│   likely near /Users/lewis/.julia/packages/WebIO/7rrhi/deps/build.jl:55
│ ┌ Warning: `replace(s::AbstractString, pat, f)` is deprecated, use `replace(s, pat => f)` instead.
│ │   caller = install_ijulia_config() at build.jl:15
│ └ @ Main ~/.julia/packages/WebIO/7rrhi/deps/build.jl:15
│ ERROR: LoadError: MethodError: no method matching dir()
│ Closest candidates are:
│   dir(!Matched::String, !Matched::AbstractString...) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/Pkg/src/API.jl:454
│ Stacktrace:
│  [1] install_ijulia_config() at /Users/lewis/.julia/packages/WebIO/7rrhi/deps/build.jl:17
│  [2] top-level scope at none:0
│  [3] include at ./boot.jl:317 [inlined]
│  [4] include_relative(::Module, ::String) at ./loading.jl:1038
│  [5] include(::Module, ::String) at ./sysimg.jl:29
│  [6] include(::String) at ./client.jl:398
│  [7] top-level scope at none:0
│ in expression starting at /Users/lewis/.julia/packages/WebIO/7rrhi/deps/build.jl:55
└ @ Pkg.Operations /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/Pkg/src/Operations.jl:1068
```

**Comment by rdeits** [comment]
Will be fixed in #169
